packages/plugins/typescript/react-apollo/src/visitor.ts

```diff
@@ -35,6 +35,10 @@ export interface ReactApolloPluginConfig extends ClientSideBasePluginConfig {
   hooksSuffix: string;
 }
 
+function hasRequiredVariables(node: OperationDefinitionNode): boolean {
+  return node.variableDefinitions?.some(variableDef => variableDef.type.kind === Kind.NON_NULL_TYPE) ?? false;
```

Comment:
A non-null variable can also have a default value.

```graphql
query HeroNameConditionalInclusion($episode: Episode, $includeName: Boolean! = true) {
  hero(episode: $episode) {
    name @include(if: $includeName)
  }
}
```

should generate the following:

```ts
export function refetchHeroNameConditionalInclusionQuery(variables?: HeroNameConditionalInclusionQueryVariables) {
  return { query: HeroNameConditionalInclusionDocument, variables: variables };
}
```

Can you please add test cases for this and adjust the implementation accordingly? 😇

Reply:
Done. Good catch.